<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="ESWS7992" failures="1" tests="2" name="Default test(failed)" time="0.078" errors="0" timestamp="2023-10-04T07:37:03 AEDT">
  <testcase classname="demo.TestNGRetryFailedDemo" name="test3" time="0.018">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="demo.TestNGRetryFailedDemo" name="test2" time="0.004"/>
  <testcase classname="demo.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
